packages/contracts-bedrock/src/cannon/PreimageOracle.sol

```diff
@@ -12,7 +11,7 @@ import "src/cannon/libraries/CannonTypes.sol";
 /// @notice A contract for storing permissioned pre-images.
 /// @custom:attribution Solady <https://github.com/Vectorized/solady/blob/main/src/utils/MerkleProofLib.sol#L13-L43>
 /// @custom:attribution Beacon Deposit Contract <0x00000000219ab540356cbb839cbe05303d7705fa>
-contract PreimageOracle is IPreimageOracle, ISemver {
+contract PreimageOracle is IPreimageOracle {
```

Comment:
Why do we remove `ISemver`?

Reply:
`IPreimageOracle` has `ISemver` on it already